infra/src/main/kotlin/fi/fta/geoviite/infra/linking/switches/SwitchFittingService.kt

```diff
@@ -611,17 +653,25 @@ fun getSharedSwitchJoint(switchStructure: SwitchStructure): Pair<SwitchJoint, Li
     return sharedSwitchJoint to switchAlignmentsContainingCommonJoint
 }
 
+data class SuggestedSwitchJointScore(
+    val jointNumber: JointNumber,
+    val jointScore: Double,
+    val farthestJointExtraScore: Double,
+) {
+    fun sum() = jointScore + farthestJointExtraScore
+}
+
```

Comment:
Puhtaasti debuggaussyistä tehty luokka, mutta ei varmaan iso synti jättää koodikantaan, tämän avulla kun tuli nimittäin jo yksi bugi korjattua, kun näki, mistä vaihde-ehdotuksen pisteet oli peräisin.